### PR TITLE
#trivial MOBILE-1027 Render Help Text on all needed fields

### PR DESCRIPTION
**Pull request by mclomello**
@dgarciasarai @victorg1991

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/BaseDDLFieldTextView.java [hunk 1]
```diff
@@ -40,7 +40,8 @@
     implements DDLFieldViewModel<T>, TextWatcher {
 
     protected TextView labelTextView;
-    protected EditText textEditText;
+	protected TextView hintTextView;
```
Comment (dgarciasarai) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/BaseDDLFieldTextView.java]: please be aware of indentation, run detekt and the end of the PR, thanks again :)

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldRadioView.java [hunk 2]
```diff
@@ -82,6 +83,11 @@ public void setField(SelectableOptionsField field) {
             radioGroup.setOrientation(HORIZONTAL);
         }
 
+        if (this.field.getTip() != null && !this.field.getTip().isEmpty()) {
+			hintTextView.setText(this.field.getTip());
```
Comment (dgarciasarai) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldRadioView.java]: same here

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/BaseDDLFieldTextView.java [hunk 3]
```diff
@@ -108,6 +109,11 @@ public void setupFieldLayout() {
         if (this.field.getPlaceHolder() != null && !this.field.getPlaceHolder().isEmpty()) {
             textEditText.setHint(this.field.getPlaceHolder());
         }
+
+        if (this.field.getTip() != null && !this.field.getTip().isEmpty()) {
```
Comment (victorg1991) [inline android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/BaseDDLFieldTextView.java]: This seems like a perfect case for a kotlin extension, don't you think? this.field.getTip().isNullOrEmpty() :)
(just and idea)

**Comment by victorg1991** [comment]
👏

File: android/viewsets/lexicon/src/main/res/values/colors_lexicon.xml [hunk 4]
```diff
@@ -18,6 +18,7 @@
     <color name="textColorSecondary_lexicon">#6b6c7e</color>
     <color name="textColorTertiary_lexicon">#e7e7ed</color>
     <color name="textColorDisabled_lexicon">#a7a9bc</color>
+	<color name="textColorHelp_lexicon">#6b6c7e</color>
```
Comment (victorlaerte) [inline android/viewsets/lexicon/src/main/res/values/colors_lexicon.xml]: identation

File: android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/BaseDDLFieldTextView.java (hunk 3 above)
Comment (victorlaerte) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/BaseDDLFieldTextView.java]: if tip is String Kotlin extensions already provide it @mclomello 
```
var strNullable : String?
strNullable = "adfasdf"
strNullable.isNullOrEmpty() //This is valid
```

Comment (victorg1991) [reply android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/BaseDDLFieldTextView.java]: Yeah, you are right, I thought it was kotlin 😂

**Pull request merged**